runtimes/web/src/index.js

```diff
@@ -245,6 +254,83 @@ async function loadCartWasm () {
     window.addEventListener("keydown", onKeyboardEvent);
     window.addEventListener("keyup", onKeyboardEvent);
 
+    let usedGamepad = -1;
+
+    function processGamepad() {
+        if (usedGamepad === -1) {
+            return;
+        }
+
+        const gamepad = navigator.getGamepads()[usedGamepad];
+        if (!gamepad) {
+            return;
+        }
+
+        const buttons = gamepad.buttons;
+        const axes = gamepad.axes;
+        
+        // not all gamepads map DPAD to 12,13, 14, 15 buttos
+        const dpad = buttons.length > 12;
```

Comment:
Would it be better to instead return early if `gamepad.mapping != "standard"`?

Reply:
This is unsafe.
mapping can be non-standart but a buttons exist and back.

Then you suggest prevent support a gamepads like this:)

![image](https://user-images.githubusercontent.com/12572222/134110998-9093b7a5-5e3d-4f41-b38d-26635a7f7567.png)

Reply:
If the mapping is non-standard, how do we know that `gamepad.buttons[14]` means "left"? The handling seems to already rely on the standard mapping.

But anyways, this is minor, we can fix it up later if necessary.